
{'date': 'Wed Oct 02 2019 08:54:52.281 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 3
  cidade = "http://www.trabalhosescolares.net/wp-content/uploads/2019/08/cidade_trabalhos_escolares.jpg"
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Oct 02 2019 09:19:44.633 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  elemento = Elemento (img = carro
                                     ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Oct 02 2019 09:19:55.932 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 9
  elemento = Elemento (img = carro ,
                                                     ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Oct 02 2019 09:26:08.611 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 4
  carro = 'https://www.pinclipart.com/picdir/middle/106-1066670_audi-clipart-fast-car-lamborghini-de-lado-png.png']
           ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Oct 02 2019 09:26:46.204 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 7
  def cidade_legal
                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Oct 02 2019 09:34:31.104 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 13
    cidade_legal()    
  module <module> line 8
    	cenario = cena (img = cidade)
NameError: name 'cena' is not defined
'''},
{'date': 'Wed Oct 02 2019 10:21:46.76 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 19
    cidade_legal()    
  module <module> line 17
    	txthomem.vai = pessoa.vai    
AttributeError: 'str' object has no attribute 'vai'
'''},